### ISSUE TITLE
Make EmojiPicker accessible

### ISSUE THREAD

**Issue by t3chguy** [vector-im/element-web#11284]
From in-review PRs you can now get to it, and get dumped into the search box, you can access the "tab" buttons but cannot access the emoji list, tab from the search input acts as if it was the last tabstop in the document causing you to go into browser controls.

- [x] ~~Use aria tabs instead of "tab" buttons with no real semantics~~
- [ ] ~~Fix tab stops around the emoji complete, using a single tab stop for the tabs, one for search, one for resultant emojis and one for quick reactions, in each of those, arrow keys should act as navigation within that widget~~

**Comment by t3chguy** [comment]
Closing in favour of https://github.com/vector-im/element-web/issues/17189
